app/src/main/java/com/example/nvdovin/weatherapp/adapter/RecyclerViewAdapter.java

```diff
@@ -88,7 +86,7 @@ private int setWeatherIcon(String id, TextView iconTextView) {
         int resID = weatherCodesMap.getBackgroundResById(id);
         String icon = context.getString(weatherCodesMap.getIconByID(id));
 
-        if (Build.VERSION.SDK_INT >= MARSHMALLOW_VERSION)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N)
```

Comment:
if you are using if else clause then you have to use braces, please pay attention on this.